### ISSUE TITLE
catalogProductAttributeRepository does not return frontend_labels

### ISSUE THREAD

**Issue by detco-monacor** [magento/magento2#8237]
Calling GET on catalogProductAttributeRepository does not return frontend-labels

### Preconditions

1. Magento EE 2.1.2
2. PHP 7.0.14 (dotdeb: 7.0.14-1~dotdeb+8.1)
3. Mysql 5.6.30 (Debian-Jessie-backports: 5.6.30-1~bpo8+1)

### Steps to reproduce

1. I have a product-attribute with the code "test_api", with frontend-labels, see backend-screenshot
2. `curl -X GET --header "Accept: application/json" --header "Authorization: Bearer <API-Token>" "https://<myserver>/rest/V1/products/attributes/test_api"`
![image](https://cloud.githubusercontent.com/assets/24474289/22207049/ec22a22a-e17d-11e6-98b8-e16aa26956a0.png)

### Expected result

return json with `"frontend_labels": [{"store_id": 1, "label": "Test 3 über API"},{"store_id":2,"label":"testing with API"}]`

### Actual result

```json
{"is_wysiwyg_enabled":false,"is_html_allowed_on_front":false,"used_for_sort_by":false,"is_filterable":false,"is_filterable_in_search":false,"is_used_in_grid":false,"is_visible_in_grid":false,"is_filterable_in_grid":false,"position":0,"apply_to":[],"is_searchable":"0","is_visible_in_advanced_search":"0","is_comparable":"0","is_used_for_promo_rules":"0","is_visible_on_front":"0","used_in_product_listing":"0","is_visible":true,"scope":"global","attribute_id":179,"attribute_code":"test_api","frontend_input":"text","entity_type_id":"4","is_required":false,"options":[],"is_user_defined":true,"default_frontend_label":"Test \u00fcber API","frontend_labels":null,"backend_type":"varchar","is_unique":"0","validation_rules":[]}
```
(note the `"frontend_labels":null,`)

**Comment by NotAndD** [comment]
I am no expert but I'm experimenting a lot with rest api for a project at work. Even if I can't reproduce your error right now, I just want to suggest that you try a fast thing, to ask the GET request with store view `all`. Something like this:
`https://<myserver>/rest/all/V1/products/attributes/test_api`

Does it change anything?

**Comment by detco-monacor** [comment]
Unfortunatly not. I tried "/rest/default/V1" (as swagger generates them), "/rest/all/V1" and "/rest/V1".
I even tried "/rest/en/V1" (i.e. the store-code of the second store-view). All of them return the same json with frontend_labels: null

**Comment by NotAndD** [comment]
I tried, since I need this functionality for my project also and the problem is there even in Magento 2.1.3.

Also found a similar issue: #3430

**Comment by misha-kotov** [comment]
Thanks for reporting the issue. I was able to reproduce and filed internal ticket MAGETWO-64315.

**Comment by magento-engcom-team** [comment]
@detco-monacor, thank you for your report.
We've created internal ticket(s) MAGETWO-64315 to track progress on the issue.

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by ghost** [comment]
HI @detco-monacor thank you for you report, this issue has already fixed in 2.3-develop branch and available on 2.3.0 release
![selection_286](https://user-images.githubusercontent.com/40756439/51592102-76c6b480-1ef7-11e9-8e4d-98f3a6daaeda.png)
